$NetBSD: patch-distbb__chroot.in,v 1.1 2012/10/05 17:14:54 cheusov Exp $

# Fix for ruby193-base failure, which uses SUDO variable internally
--- distbb_chroot.in.orig	2012-09-02 19:24:36.000000000 +0000
+++ distbb_chroot.in
@@ -32,4 +32,4 @@ fi
 
 env="$env DISTBB_CONF=$DISTBB_CONF"
 
-$SUDO chroot $root$id su - root -c "env $env $*"
+$DISTBB_SUDO chroot $root$id su - root -c "env $env $*"
